geth/jail/jail_test.go

```diff
@@ -115,6 +116,44 @@ func (s *JailTestSuite) TestFunctionCall() {
 	require.Equal(expectedResponse, response)
 }
 
+func (s *JailTestSuite) TestJailRPCAsyncSend() {
+	require := s.Require()
+	var wg sync.WaitGroup
```

Comment:
Can we move `wg` closer to where it's used? @adambabik already noted that variables should be declared as close as possible to the place they're used and I totally agree with that.